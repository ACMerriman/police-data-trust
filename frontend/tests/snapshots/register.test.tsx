import * as React from "react"
import Register from "../../pages/register"
import renderer from "react-test-renderer"

it("renders Register correctly", () => {
  const tree = renderer.create(<Register />).toJSON()
  expect(tree).toMatchSnapshot()
})
