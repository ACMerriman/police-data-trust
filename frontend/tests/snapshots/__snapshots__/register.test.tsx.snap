// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Register correctly 1`] = `
<div
  className="Register"
>
  I AM REGISTRATION!!!!!!
</div>
`;
